run_tests.sh

```diff
@@ -25,6 +25,9 @@ then
     pytest --cov-report=xml --cov=art --cov-append  -q -vv tests/defences/detector/poison/test_spectral_signature_defense.py --framework=$framework --durations=20 --durations-min=0
     if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed defences/detector/poison/test_spectral_signature_defense.py tests"; fi
 
+    pytest --cov-report=xml --cov=art --cov-append  -q -vv tests/defences/detector/poison/test_clustering_centroid_analysis.py --framework=$framework --durations=20 --durations-min=0
+    if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed defences/detector/poison/test_clustering_centroid_analysis.py tests"; fi
+
```

Comment:
```suggestion
```
Replace this with a new line after this line ""tests/defences/detector/poison/test_clustering_analyzer.py" \" around line 156.